interface-definitions/pki.xml.in

```diff
@@ -0,0 +1,203 @@
+        </children>
+      </tagNode>
+      <tagNode name="certificate">
+        <properties>
+          <help>Certificate</help>
+        </properties>
+        <children>
+          <leafNode name="certificate">
+            <properties>
+              <help>Certificate in PEM format</help>
+            </properties>
+          </leafNode>
+          <leafNode name="description">
+            <properties>
+              <help>Description</help>
+            </properties>
+          </leafNode>
+          <node name="private">
+            <properties>
+              <help>Certificate private key</help>
+            </properties>
+            <children>
+              <leafNode name="key">
+                <properties>
+                  <help>Certificate private key in PEM format</help>
+                </properties>
+              </leafNode>
+              <leafNode name="password-protected">
+                <properties>
+                  <help>Certificate private key is password protected</help>
+                  <valueless/>
+                </properties>
+              </leafNode>
+            </children>
+          </node>
+          <leafNode name="revoke">
+            <properties>
+              <help>If CA is present, this certificate will be included in generated CRLs</help>
+              <valueless/>
+            </properties>
+          </leafNode>
+        </children>
+      </tagNode>
+      <tagNode name="dh">
+        <properties>
+          <help>Diffie-Hellman parameters</help>
+        </properties>
+        <children>
+          <leafNode name="parameters">
+            <properties>
+              <help>DH parameters in PEM format</help>
+            </properties>
+          </leafNode>
+        </children>
+      </tagNode>
+      <tagNode name="key-pair">
+        <properties>
+          <help>Public and private keys</help>
+        </properties>
+        <children>
+          <node name="public">
+            <properties>
+              <help>Public key</help>
+            </properties>
+            <children>
+              <leafNode name="key">
+                <properties>
+                  <help>Public key in PEM format</help>
+                </properties>
+              </leafNode>
+            </children>
+          </node>
+          <node name="private">
+            <properties>
+              <help>Private key</help>
+            </properties>
+            <children>
+              <leafNode name="key">
+                <properties>
+                  <help>Private key in PEM format</help>
+                </properties>
+              </leafNode>
+              <leafNode name="password-protected">
+                <properties>
+                  <help>Private key is password protected</help>
+                  <valueless/>
+                </properties>
+              </leafNode>
+            </children>
+          </node>
+        </children>
+      </tagNode>
+      <node name="openvpn">
```

Comment:
What about an OpenVPN Site2Site key aka `shared-secret-key-file` CLI option?

Reply:
Do you think its best if I just rename the tls-auth node to a common name of `static-key`? Then in a future PR where OpenVPN is integrated we can reference that single node for both `shared-secret-key` and `tls-auth`.

Reply:
I'd simply call it `shared-secret`, as that is what OpenVPN calls it internally:

```
--genkey        : Generate a random key to be used as a shared secret,
                  for use with the --secret option.
```

Reply:
Changed